looks like CI isn't happy with one of the tests added in this PR

```
 FAIL  test_localhost_rejection (fossunited.foss_hackathon.doctype.foss_hackathon_participant.test_foss_hackathon_participant.TestFOSSHackathonParticipant)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/frappe-bench/apps/fossunited/fossunited/foss_hackathon/doctype/foss_hackathon_participant/test_foss_hackathon_participant.py", line 111, in test_localhost_rejection
    with self.assertRaises(frappe.PermissionError):
    self = <fossunited.foss_hackathon.doctype.foss_hackathon_participant.test_foss_hackathon_participant.TestFOSSHackathonParticipant testMethod=test_localhost_rejection>
AssertionError: PermissionError not raised
```

--- a/fossunited/foss_hackathon/doctype/foss_hackathon_participant/test_foss_hackathon_participant.py
+++ b/fossunited/foss_hackathon/doctype/foss_hackathon_participant/test_foss_hackathon_participant.py
@@ -37,6 +37,7 @@ class TestFOSSHackathonParticipant(IntegrationTestCase):
     def tearDown(self):
         frappe.set_user("Administrator")
         self.participant_1.delete(force=True)
+        self.localhost.delete(force=True)
         self.hackathon.delete(force=True)
         self.chapter.delete(force=True)
 
@@ -107,6 +108,8 @@ class TestFOSSHackathonParticipant(IntegrationTestCase):
         frappe.set_user(self.PARTICIPANT_1)
         self.participant_1.wants_to_attend_locally = 1
         self.participant_1.localhost = self.localhost.name
+
+        self.assertEqual(self.participant_1.localhost_request_status, "Rejected")
         # Then a permission error should be thrown
         with self.assertRaises(frappe.PermissionError):
             self.participant_1.save()

--- a/fossunited/foss_hackathon/doctype/foss_hackathon_participant/test_foss_hackathon_participant.py
+++ b/fossunited/foss_hackathon/doctype/foss_hackathon_participant/test_foss_hackathon_participant.py
@@ -13,7 +13,7 @@ from fossunited.tests.utils import (
 class TestFOSSHackathonParticipant(IntegrationTestCase):
     def setUp(self):
         self.LOCALHOST_ORGANIZER = "test3@example.com"
-        self.PARTICIPANT_1 = "test1@example.com"
+        self.PARTICIPANT_1 = "test4@example.com"
         self.PARTICIPANT_2 = "test2@example.com"
         self.chapter = insert_test_chapter()
         self.hackathon = insert_test_hackathon(chapter=self.chapter.name)

